### PR TITLE
[NCLOUD-926] Add screenshots

### PR DESCRIPTION
**Pull request by dscodi**
Added screenshots based on https://networktocode.com/apps/capacity-metrics/

### CODE REVIEW

File: docs/images/capacity-metrics-screenshot-04.png
Comment (jvanderaa) [inline docs/images/capacity-metrics-screenshot-04.png:1]: Would be better if we went with the documentation prefixes.

Comment (dscodi) [reply docs/images/capacity-metrics-screenshot-04.png:1]: I'm sorry, what do you mean. Can you give an example?

**Review by davidban77 — APPROVED** [review]
LGTM

File: docs/images/capacity-metrics-screenshot-04.png
Comment (jvanderaa) [reply docs/images/capacity-metrics-screenshot-04.png:1]: 192.0.2.0/24, 203.0.113.0/24, 198.51.100.0/24

https://www.auvik.com/franklyit/blog/special-ip-address-ranges/

@dscodi

Comment (jvanderaa) [reply docs/images/capacity-metrics-screenshot-04.png:1]: Not a stopper, but should use these ranges when we can in documentation.

Comment (whitej6) [reply docs/images/capacity-metrics-screenshot-04.png:1]: @jvanderaa I agree that going forward all IPs should be in the documentation range, has this been documented/discussed as part of dev-standards?

**Pull request merged**